### PR TITLE
Initial version of the Load Balancer taxonomy

### PR DESCRIPTION
**Pull request by cfisher-scottlogic**
Adds Cloud Load Balancer to CCC service taxonomy as per #133.

### CODE REVIEW

**Review by AdrianHammond — COMMENTED** [review]
@cfisher-scottlogic   Hi Cara. - Re line 53 - CCC-050335 - this looks to specific capability for AWS Outpost? I think the Taxonomy should be vendor agnostic so I think we should update / remove this line, what do you think? Many thanks Adrian

File: services/networking/loadbalancer/taxonomy.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+| CCC-050334  | Traffic splitting / weighted routing                  | Can distribute incoming traffic across multiple backend resources based on predefined weights or percentages. (e.g. for canary deployments, A/B testing, blue-green deployments, or gradual traffic migrations)                                                                                                                               |
+| CCC-050335  | Outpost support                                       | Ability to extend cloud-provisioned load balancing services to on-premises or edge locations using dedicated hardware or software appliances (outposts).                                                                                                                                                                                      |
```
Comment (mlysaght2017) [inline services/networking/loadbalancer/taxonomy.md:53]: Looks vendor specific

**Review by mlysaght2017 — COMMENTED** [review]
Very comprehensive, nice job

File: services/networking/loadbalancer/taxonomy.md (hunk 1 above)
Comment (AdrianHammond) [reply services/networking/loadbalancer/taxonomy.md:53]: Thats works for me @cfisher-scottlogic

Comment (cfisher-scottlogic) [reply services/networking/loadbalancer/taxonomy.md]: Sorted! Also changed description slightly: [ea952e0](https://github.com/finos/common-cloud-controls/pull/149/commits/ea952e0f4b59cc62932bc59071017c3069b91cf6).

File: services/networking/loadbalancer/taxonomy.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ use in financial services ecosystems.
...
-| CCC-050335  | Outpost support                                       | Ability to extend cloud-provisioned load balancing services to on-premises or edge locations using dedicated hardware or software appliances (outposts).                                                                                                                                                                                      |
+| CCC-050335  | Hybrid cloud connectivity                             | Can extend cloud-provisioned load balancing services across on-premises, private, and other public cloud infrastructures.                                                                                                                                                                                                                     |
```
Comment (AdrianHammond) [inline services/networking/loadbalancer/taxonomy.md:53]: LGTM

**Review by AdrianHammond — COMMENTED** [review]
This LGTM  - not certain if its okay for me to merge. @mark-rushing thoughts?

**Review by markrushing — APPROVED** [review]
Agreed and approved

**Review by AdrianHammond — APPROVED** [review]
Merging post discussion on all project meeting on 04/04.

**Pull request merged**